src/test/java/org/fcrepo/importexport/integration/ImporterIT.java

```diff
@@ -279,6 +279,33 @@ public void testImportMappedTrailingSlash() throws Exception {
         assertTrue(resourceExists(childURI));
     }
 
+    @Test
+    public void testRepositoryRoot() throws Exception {
```

Comment:
For the most part, I'd say that's true — I've pushed a commit to move the second test to the unit test, and added a few others handling the main cases with mocks.  But I'd also like to keep the one test that verifies that we are finding the correct root of the repository we're running our integration tests against.